__tests__/app.test.js

```diff
@@ -42,15 +41,49 @@ describe('GET requests', () => {
 
     describe("GET /api", () => {
         test("Status: 200 - Should respond with a JSON object that describes all of the availlable endpoints", () => {
-          return request(app)
-          .get("/api")
-          .expect(200)
-          .then(({body}) => {
-            const {endpoints} = body;
-            expect(typeof endpoints).toBe('object')
-            expect(Object.entries(endpoints).length).toBe(Object.entries(endpointsCopy).length)
+            return request(app)
+            .get("/api")
+            .expect(200)
+            .then(({body}) => {
+                const {endpoints} = body;
+                expect(endpoints).toEqual(endpointsCopy)
             })
-          })
         })
+    })
+
+    describe('GET /api/articles/:article_id', () => {
+        test('200 - should respond with an object containing the correct properties', () => {
+            return request(app)
+            .get("/api/articles/4")
+            .expect(200)
+            .then(({body}) => {
+                expect(typeof body.article).toBe('object');
+                expect(body.article).toHaveProperty("author", expect.any(String))
+                expect(body.article).toHaveProperty("title", expect.any(String))
+                expect(body.article).toHaveProperty("article_id", expect.any(Number))
+                expect(body.article).toHaveProperty("body", expect.any(String))
+                expect(body.article).toHaveProperty("topic", expect.any(String))
+                expect(body.article).toHaveProperty("created_at", expect.any(String))
+                expect(body.article).toHaveProperty("votes", expect.any(Number))
```

Comment:
This should be the specific article content e.g. the article number should be 4, the title should be as it is when returned, etc

Reply:
You may also want to look into a couple of Jest matchers for objects - toMatchObject and objectContaining - to avoid breaking this test later when we add another key-value pair